internal/utils/miscellaneous.go

```diff
@@ -8,7 +8,9 @@ import (
 	"fmt"
 	"maps"
 	"strings"
+	"time"
 
+	cron "github.com/robfig/cron/v3"
```

Comment:
We don't need to add `cron` as the package shorthand, as versions are not package names.